app/models/profile.rb

```diff
@@ -69,5 +72,23 @@ def from_openscap_parser(op_profile, existing: nil, benchmark_id: nil, account_i
 
       profile
     end
+
+    # Transforming value keys from ref_id to uuid
+    def prepare_values(values)
+      return nil unless values
+
+      value_definitions = ValueDefinition.where(ref_id: values.keys).index_by(&:ref_id)
+
+      raise ActiveRecord::RecordNotFound unless value_definitions.count == values.count
+
+      values.transform_keys { |key| value_definitions[key].id }
+    end
+  end
```

Comment:
@skateman I'm confused why we need this because we're already storing the `value_overrides` using value uuids like `{uuid, value}` 🤔

Reply:
The frontend is working with `ref_id` instead of `uuid`, eventually I want both APIs to work with both (like tailoring).

Reply:
Ah okay, that makes sense now